python/examples/example_jax/ExampleJax.ipynb

```diff
@@ -0,0 +1,1087 @@
+      "ld: warning: -undefined dynamic_lookup may not work with chained fixups\n",
+      "\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2023-02-16 12:37:31.673 - amici.petab_import - INFO - Finished Importing PEtab model                (1.36E+01s)\n",
+      "2023-02-16 12:37:31.684 - amici.petab_import - INFO - Successfully loaded model Boehm_JProteomeRes2014 from /Users/fabian/Documents/projects/AMICI/documentation/amici_models/Boehm_JProteomeRes2014.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from amici.petab_import import import_petab_problem\n",
+    "amici_model = import_petab_problem(petab_problem, force_compile=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7827aaf7",
+   "metadata": {},
+   "source": [
+    "# JAX implementation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e2ef051a",
+   "metadata": {},
+   "source": [
+    "For full jax support, we would have to implement a new [primitive](https://jax.readthedocs.io/en/latest/notebooks/How_JAX_primitives_work.html), which would require quite a bit of engineering, and in the end wouldn't add much benefit since AMICI can't run on GPUs. Instead will interface AMICI using the experimental jax module [`host_callback`](https://jax.readthedocs.io/en/latest/jax.experimental.host_callback.html). "
```

Comment:
```suggestion
    "For full jax support, we would have to implement a new [primitive](https://jax.readthedocs.io/en/latest/notebooks/How_JAX_primitives_work.html), which would require quite a bit of engineering, and in the end wouldn't add much benefit since AMICI can't run on GPUs. Instead will interface AMICI using the experimental jax module [host_callback](https://jax.readthedocs.io/en/latest/jax.experimental.host_callback.html). "
```